boot.php

```diff
@@ -498,36 +497,6 @@
 if (!defined('CURLE_OPERATION_TIMEDOUT')) {
 	define('CURLE_OPERATION_TIMEDOUT', CURLE_OPERATION_TIMEOUTED);
 }
-/**
```

Comment:
Why is this removed? Does not seem to be related to the config format change.

Reply:
It's been integrated in `App::__construct()`. The magic quotes setting has been removed from PHP in version 5.3, so I removed it entirely since we require PHP 5.6.1 now.